src/ToricVarieties/ToricDivisors.jl

```diff
@@ -71,7 +71,7 @@ export DivisorOfCharacter
 ######################
 
 @doc Markdown.doc"""
-    iscartier(d::ToricDivisor)
+    iscartier(td::ToricDivisor)
 
 Checks if the divisor `d` is Cartier.
```

Comment:
```suggestion
Checks if the divisor `td` is Cartier.
```

Reply:
Yes, this is also fixed in the version that I will force-push in a few minutes.